maxim/logger/crewai/client.py

```
@@ -889,12 +922,17 @@ def maxim_wrapper(self, *args, **kwargs):
 
     # --- 0. Patch LLM._handle_non_streaming_response to capture usage ---
     if LLM is not None and hasattr(LLM, "_handle_non_streaming_response"):
-        original_handle_method = getattr(LLM, "_handle_non_streaming_response")
-        wrapper_handle = make_handle_non_streaming_wrapper(original_handle_method)
-        setattr(LLM, "_handle_non_streaming_response", wrapper_handle)
-        scribe().info(
-            "[MaximSDK] Patched crewai.LLM._handle_non_streaming_response to capture usage."
-        )
+        try:
+            original_handle_method = getattr(LLM, "_handle_non_streaming_response")
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)